sunkit_image/coalignment/__init__.py

```diff
@@ -0,0 +1,4 @@
+from sunkit_image.coalignment.interface import coalignment
+from sunkit_image.coalignment.match_template import match_template_coalign  # noqa : F401
```

Comment:
```suggestion
from sunkit_image.coalignment.match_template import match_template_coalign as _
```
See if that avoids the need for F401